include/swift/Sema/ConstraintLocator.h

```diff
@@ -211,6 +211,11 @@ class ConstraintLocator : public llvm::FoldingSetNode {
   /// Determine whether this locator points to the generic parameter.
   bool isForGenericParameter() const;
 
+  /// Determine whether this locator points to the outer
+  /// property wrapper's type for a given composed property
```

Comment:
I think this applies to every wrapped in the composition expect to the innermost one actually so e.g. `@A @B @C var ...`, this locator element would be attached to a wrapped value of `@B` and `@A`.